src/poetry/console/commands/install.py

```diff
@@ -30,6 +30,17 @@ class InstallCommand(InstallerCommand):
         option(
             "no-root", None, "Do not install the root package (the current project)."
         ),
+        option(
+            "no-directory",
+            None,
+            (
+                "Do not install any directory path dependencies (ones using `package ="
+                ' { path = "..." }`\'; useful to install dependencies without source'
```

Comment:
```suggestion
                "Do not install any directory path dependencies; useful to install dependencies without source'
```

I'd leave that away and expect a user who doesn't know what it is to not use this option or consult the documentation.